query.txt

```diff
@@ -0,0 +1,4 @@
+[out:json];
+relation(3986083);
```

Comment:
The query should be islands of Indonesia, but if that is too much data, please do islands of Kalimantan.